docs/source/api/models/lithium_ion/spm.rst

```diff
@@ -7,4 +7,6 @@ Single Particle Model (SPM)
 .. autoclass:: pybamm.lithium_ion.BasicSPM
     :members:
 
+.. autoclass:: pybamm.lithium_ion.BasicSPM_with_3DThermal
```

Comment:
Minor point, but can we change the class name to not have underscores to match the naming style elsewhere? maybe `Basic3DThermalSPM`?